src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorTestConstants.java

```diff
@@ -68,9 +70,24 @@
 
     public static final String DBSNP_HG19_SNIPPET_FILE_PATH = FUNCOTATOR_TEST_DIR + "dbSNP_hg19_snippet.vcf";
 
+    public static final String GENCODE_TRANSCRIPT_FASTA_FILE_NAME = FUNCOTATOR_LARGE_FILES_DIR + "gencode.v19.pc_transcripts.fasta";
+
     // ----------------------------------------------------------------------
     // Integration Test Variables:
+
     public static final String MUC16_MNP_FILE_BASE_NAME    = FUNCOTATOR_TEST_DIR + "MUC16_MNP";
     public static final String PIK3CA_SNP_FILE_BASE_NAME   = FUNCOTATOR_TEST_DIR + "PIK3CA_SNPS";
     public static final String PIK3CA_INDEL_FILE_BASE_NAME = FUNCOTATOR_TEST_DIR + "PIK3CA_INDELS";
+
+    // ----------------------------------------------------------------------
+    // Regression Test Variables:
+
+    public static final String REGRESSION_TEST_HG19_DATA_SET_1 = FUNCOTATOR_LARGE_FILES_DIR + "regressionTestVariantSet1.vcf";
+    public static final String REGRESSION_TEST_HG19_DATA_SET_1_EXPECTED_OUTPUT = FUNCOTATOR_LARGE_FILES_DIR + "regressionTestVariantSet1_expected.vcf";
+
+    public static final String REGRESSION_TEST_HG19_DATA_SET_2 = FUNCOTATOR_LARGE_FILES_DIR + "regressionTestVariantSet2.vcf";
```

Comment:
Can you document the provenance of "set1" vs. "set2", and mention any notable differences between them?

Reply:
Fixed.